Corale.Colore/Core/IDevice.cs

```diff
@@ -28,6 +28,8 @@
 // </copyright>
 // ---------------------------------------------------------------------------------------
 
+using System;
```

Comment:
Fix incorrect placement of using statement (use deepest possible placement).

Here: place inside namespace.